### PR TITLE
Feature - add loading spinner on upload image

### LINKED ISSUE #1614: Add loading spinner on upload image
Add a new loading spinner on `TileEditor.component` . It will be displayed when the image is loading. 
Use [progress MUI v4](https://v4.mui.com/components/progress/#progress)

image

### PR DESCRIPTION
**Pull request by rakeshkumar1019**
On this PR new loading spinner is added on`TileEditor.component `. It will be displayed when the image is loading.
Spinner is used from  [progress MUI v4](https://v4.mui.com/components/progress/#progress)

see this screenshot for the reference. 
Screenshot 2023-11-24 at 6 41 43 PM

close #1614

### CODE REVIEW

File: src/components/Board/TileEditor/TileEditor.component.js [hunk 1]
```diff
@@ -107,7 +108,8 @@ export class TileEditor extends Component {
       tile: this.defaultTile,
       linkedBoard: '',
       imageUploadedData: [],
-      isEditImageBtnActive: false
+      isEditImageBtnActive: false,
+      loading: false
```
Comment (RodriSanchez1) [inline src/components/Board/TileEditor/TileEditor.component.js]: Change the prop name to `isLoading`

**Review by RodriSanchez1 — CHANGES_REQUESTED** [review]
Hi! @rakeshkumar1019 Thank for your contribution! Your work is well! But the spinner will be not displayed when the app is proccesing the image. The inItial change of the loading state needs to be under  the `InputImage.component.js`.  Let me know if you need a further explanation 🙌

File: src/components/Board/TileEditor/TileEditor.component.js (hunk 1 above)
Comment (rakeshkumar1019) [reply src/components/Board/TileEditor/TileEditor.component.js]: Hi @RodriSanchez1  Thanks for acknowledging the pr & changes are made as requested.kindly review it.

**Comment by martinbedouret** [comment]
@RodriSanchez1  let me know if this can be merged

**Review by RodriSanchez1 — APPROVED** [review]

**Pull request merged**